packages/list/list.scss

```diff
@@ -52,4 +71,33 @@ $white-diamond: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg
             list-style: lower-alpha;
         }
     }
+
+    &__item {
+        &__iconed {
+            list-style: none;
+            position: relative;
+            &::before {
+                text-indent: -9999px;
+                position: absolute;
+                top: 0.25em;
+                left: rem(-23px);
```

Comment:
Bruker man både rem og em?

Reply:
Vi bruker rem for mer eller mindre alt. Men akkurat her er jeg ute etter oppførslen til `em` for å få elementet til å flytte seg relativt med font-størrelsen til parent.